nautobot/utilities/utils.py

```diff
@@ -931,9 +932,9 @@ def convert_querydict_to_factory_formset_acceptable_querydict(request_querydict,
 
 
 def is_single_choice_field(filterset_class, field_name):
-    # Some filter parameters do not accept multiple values, e.g DateTime fields, boolean fields, q field etc.
+    # Some filter parameters do not accept multiple values, e.g DateTime, Boolean, Int fields and the q field, etc.
     field = get_filterset_field(filterset_class, field_name)
-    return isinstance(field, (DateFilter, DateTimeFilter, TimeFilter, BooleanFilter)) or field_name == "q"
+    return not isinstance(field, django_filters.MultipleChoiceFilter) or field_name == "q"
```

Comment:
May not need the special case for `field_name == "q"` any more, since SearchFilter is not a MultipleChoiceFilter?